Source/CircleOfFifths.swift

```diff
@@ -23,33 +23,18 @@ internal enum CircleChordType {
 public class CircleOfFifths: CRView {
   public var scale = Scale(type: .minor, key: .c) { didSet { redraw() }}
 
-  #if os(OSX)
-    @IBInspectable public var defaultColor: NSColor = .white { didSet { redraw() }}
-    @IBInspectable public var highlightedColor: NSColor = .red { didSet { redraw() }}
-    @IBInspectable public var disabledColor: NSColor = .lightGray { didSet { redraw() }}
-    @IBInspectable public var textColor: NSColor = .black { didSet { redraw() }}
-    @IBInspectable public var circlePieLineColor: NSColor = .black { didSet { redraw() }}
-    @IBInspectable public var majorColor: NSColor = .red { didSet { redraw() }}
-    @IBInspectable public var minorColor: NSColor = .blue { didSet { redraw() }}
-    @IBInspectable public var diminishedColor: NSColor = .green { didSet { redraw() }}
-    @IBInspectable public var majorTextColor: NSColor = .black { didSet { redraw() }}
-    @IBInspectable public var minorTextColor: NSColor = .black { didSet { redraw() }}
-    @IBInspectable public var diminishedTextColor: NSColor = .black { didSet { redraw() }}
-    @IBInspectable public var chordPieLineColor: NSColor = .black { didSet { redraw() }}
-  #elseif os(iOS) || os(tvOS)
-    @IBInspectable public var defaultColor: UIColor = .white { didSet { redraw() }}
-    @IBInspectable public var highlightedColor: UIColor = .red { didSet { redraw() }}
-    @IBInspectable public var disabledColor: UIColor = .lightGray { didSet { redraw() }}
-    @IBInspectable public var textColor: UIColor = .black { didSet { redraw() }}
-    @IBInspectable public var circlePieLineColor: UIColor = .black { didSet { redraw() }}
-    @IBInspectable public var majorColor: UIColor = .red { didSet { redraw() }}
-    @IBInspectable public var minorColor: UIColor = .blue { didSet { redraw() }}
-    @IBInspectable public var diminishedColor: UIColor = .green { didSet { redraw() }}
-    @IBInspectable public var majorTextColor: UIColor = .black { didSet { redraw() }}
-    @IBInspectable public var minorTextColor: UIColor = .black { didSet { redraw() }}
-    @IBInspectable public var diminishedTextColor: UIColor = .black { didSet { redraw() }}
-    @IBInspectable public var chordPieLineColor: UIColor = .black { didSet { redraw() }}
-  #endif
```

Comment:
This ugly part of the code is actually necessary because of an interface builder bug. We need to do this in order to see these properties in the interface builder for both iOS and macOS.